### PR TITLE
Fix: Use Icon property for RichCommands

### PR DESCRIPTION
**Pull request by hecksmosis**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- After the changes introduced in #11526, we should use the introduced Icon property in RichCommands (Cut, Copy and Delete)

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by yaira2** [comment]
Can you fix the merge conflicts?

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -107,25 +107,23 @@
 					AccessKey="X"
 					AutomationProperties.AutomationId="InnerNavigationToolbarCutButton"
 					Command="{x:Bind Commands.CutItem, Mode=OneWay}"
+          Content="{x:Bind Commands.CutItem.Icon}"
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:110]: Spacing looks off, can you run xaml styler?

Comment (hecksmosis) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:110]: This was the github merge, i'll fix it

Comment (yaira2) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml]: Spacing still looks off.

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 2]
```diff
@@ -207,7 +201,7 @@
 					x:Load="{x:Bind ViewModel.InstanceViewModel.IsPageTypeRecycleBin, Mode=OneWay, FallbackValue=False}"
 					AccessKey="B"
 					Command="{x:Bind Commands.EmptyRecycleBin}"
-					Content="{x:Bind Commands.EmptyRecycleBin.Icon}"
+					Content="{x:Bind Commands.EmptyRecycleBin.OpacityIcon}"
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:204]: Isn't this supposed to be `Icon` now?

Comment (yaira2) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:204]: @cinqmilleans

Comment (hecksmosis) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:204]: Yes, I'll implement it

Comment (hecksmosis) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:204]: I think it was a bad merge because @cinqmilleans changed it in his pr

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**